dmipy/core/modeling_framework.py

```diff
@@ -27,6 +28,7 @@
 from ..optimizers.multi_tissue_convex_optimizer import (
     MultiTissueConvexOptimizer)
 from dipy.utils.optpkg import optional_package
+from dmipy.utils.build_sphere import get_hemisphere
```

Comment:
You didn't say why you wanted to use this new function for getting a hemisphere?

Reply:
true story. didn't notice that before. We can definitely use that one...

Reply:
https://github.com/AthenaEPI/dmipy/blob/c246fa7842dfb8d4a114e82f9a7efe0743dabaa7/dmipy/optimizers_fod/csd_cvxpy.py#L77